testing/regress/ecl/translatedisk.ecl

```diff
@@ -25,6 +25,7 @@ multiPart := #IFDEFINED(root.multiPart, false);
 
 //--- end of version configuration ---
 
+#onwarning(2036, ignore);
```

Comment:
Also need to ignore 4522?

Reply:
Looks like it - for some reason it seems only the first warning gets flagged as a difference by regression suite each time?